src/python/ServerUtilities.py

```diff
@@ -248,3 +256,19 @@ def mostCommon(lst, default=0):
     except ValueError:
         return default
 
+
+def getTimeFromTaskname(taskname):
+    """ Get the submission time from the taskname and return the seconds since epoch
+        corresponding to it
+    """
+
+    #validate taskname. In principle not necessary, but..
+    if not isinstance(taskname, str):
+        raise TypeError('In ServerUtilities.getTimeFromTaskname: "taskname" parameter must be a string')
+    stime = taskname.split(':')[0] #s stands for string
+    stimePattern = '^\d{6}_\d{6}$'
```

Comment:
Regexp is for REST regular expressions. That is why it is in CRABInterface, it is used for the validate functions. That's why I did not add it there.